db/migrate/20210819182409_create_reservations.rb

```diff
@@ -5,6 +5,7 @@ def change
     create_table :reservations do |t|
       t.references :screening, null: false, foreign_key: true
       t.references :status, null: false, foreign_key: true
+      t.references :user, null: false, foreign_key: true
```

Comment:
You should not change existing migrations - if it was already run those changes will not be applied, and if it was not, then probably `users` table does not exist and it will return an error. You would need to create new migrations after migration for user model that would just change this table.

Reply:
When you put it like this it makes perfect sense, but I was under an impression it can _pass_ when in development.
Especially since I got green light when I did it [last time](https://github.com/katemildner/Monte-Cinema/pull/4/commits/3075009c7adb1d141b4aa0faf84cf5a81fcfbea2). 🤔 
But I will not do this again, for the sake of good practices. 💪🏻 

I will revert this commit and do it properly.